src/components/List/List.js

```diff
@@ -0,0 +1,23 @@
+import { RemoveButton } from "../RemoveButton/RemoveButton";
+
+export function List({ elements, section, handleRemoveItem }) {
+  const list = elements.map((element, index) => {
+    let checkboxes = [];
```

Comment:
If you declare variable with an empty array you can use 'const'.

Reply:
done